### PR TITLE
Fixed missing scriptEnd at the end of Html::initEditorSystem function

### PR DESCRIPTION
**Pull request by tomolimo**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: inc/html.class.php [hunk 1]
```diff
@@ -3920,9 +3920,9 @@ static function initEditorSystem($name, $rand = '', $display = true, $readonly =
       });";
 
       if ($display) {
-         echo  Html::scriptBlock($js);
+         echo  Html::scriptEnd($js);
```
Comment (trasher) [inline inc/html.class.php]: `scriptEnd` takes no arguments.

I'm not sure what is the best between your proposal, and keep `scriptBlock` and remove `scriptStart`. @orthagh ?

Comment (tomolimo) [reply inc/html.class.php]: I agree, the question is real, but we should choose one or the other, but of course not a mix :)
It seems that I've been a bit too quick :)

Comment (trasher) [reply inc/html.class.php]: Huh and by the way... With current proposal, `$js` is never sent to the buffer; it should not works

Comment (trasher) [reply inc/html.class.php]: Looking at the code; `scriptBlock` is used about 69 times, where `scriptStart|End` is used only twice.

**Comment by tomolimo** [comment]
you're welcome

**Pull request merged**